prime-router/metadata/hl7_mapping/STLTs/CA/CA.yml

```diff
@@ -131,7 +108,6 @@ elements:
     resource: 'Bundle'
     value: [ '"PRIME ReportStream"' ]
 
-  # The transforms below could be moved to the sender transform/enrichment step for simple report once FHIR transforms are ready
```

Comment:
Guessing we decided not to move the below elements into the sender transforms?

Reply:
Yes, Victor said they should not be removed, so I stopped work on that and removed the comment.